WalletWasabi.Fluent/Converters/HardwareWalletIconConverter.cs

```diff
@@ -25,7 +25,7 @@ private HardwareWalletIconConverter()
 				{
 					case Hwi.Models.HardwareWalletModels.Coldcard:
 					case Hwi.Models.HardwareWalletModels.Coldcard_Simulator:
-						uri = new ("avares://WalletWasabi.Fluent/Assets/HardwareIcons/coldcard.png");						
+						uri = new ("avares://WalletWasabi.Fluent/Assets/HardwareIcons/coldcard.png");
```

Comment:
these whitespace changes autoformatted it seems...

Reply:
btw this converter is not used anywhere, it can be deleted